mammoth.repos

```diff
@@ -18,8 +18,11 @@ repositories:
   serial_ros2:
     type: git
     url: https://github.com/iscumd/isc_serial.git
-    version: ros2   
   serial_utils_ros2:
     type: git
     url: https://github.com/iscumd/isc_serial_utils.git
-    version: ros2   
\ No newline at end of file
+    version: ros2     
+  ros2_ouster:
```

Comment:
this should be an apt package, add it to cmakelists and package.xml and install using apt

Reply:
fixed